### ISSUE TITLE
Error: Undefined operation: "320px times 0".

### ISSUE THREAD

**Issue by markhughes** [seavor/vunits-sass-poly#2]
So I have imported your library and then I have this:

``` scss
    #container {
      @include vu-unit(width, 90vw);
    }
```

But when sass tries to compile I get thrown with this error:

```
    Running "sass:dist" (sass) task
    Error: Undefined operation: "320px times 0".
            on line 2 of bower_components/vuntis-sass-poly/sass/vunits.scss, in `vu-compile'
            from line 166 of bower_components/vuntis-sass-poly/sass/vunits.scss
            from line 16 of public/assets/sass/main.scss
```

Line 16 is

``` scss
    @import "../../../bower_components/vuntis-sass-poly/sass/vunits";
```

Line 166 is calling

``` scss
    @include vu-compile();
```

And line 2 is

``` scss
    @return $input / ($input * 0 + 1); // <= Sass bug?
```

Maybe I'm doing something wrong? :) Let me know if I can provide any more information

**Comment by seavor** [comment]
Hey, so i actually haven't touched this code in quite awhile, it was a personal sideproject that I never got a chance to complete. You have the general flow correct, you import the library at the top of your sass file, include v-units as you need them, and run the compile function at the end. The compiler logic is incomplete tho, and I was still in the process of trying to figure out how the compiler should run. There's an issue where the selectors don't respect media queries, making the library unusable for responsive design. The last thing i think i was trying to do was have the compiler just input the back up property/unit pair right behind the v-unit, instead of appending a whole extra set of selectors and properties.

The issue you're facing is probably a lil more basic tho, somehow we're asking sass to compile a number times 0, which is illogical and will throw an error. If you feel up to it, i encourage you to fiddle around with the logic and see if you can't get it working. I may be able to get around to finishing this, but it will be a couple months before i probably find the time, and so any contributions would be helpful.

**Comment by seavor** [comment]
Correction (remembering what I was doing now): The times by 0 function is a logic trick to get the number split from the unit (e.g. 320px / 1px = 320). For some reason Sass doesn't like it.

**Comment by seavor** [comment]
I suspect it's these 2 lines which are to blame, but not sure what the issue is yet. The strip-unit function allows v-units in the logic trick, but not the pixel units defined in the device breakpoints.

$calced-width: floor(strip-unit($width) \* $ratio);
$calced-height: floor(strip-unit($height) \* $ratio);

**Comment by seavor** [comment]
It actually seems to be these 2 lines in the vu-compile function:

#{$property}: vu-calc($value, $query-width, $query-height);
#{$property}: vu-calc($value, $query-width, $query-height);

**Comment by seavor** [comment]
Or maybe these 2:

#{$property}: vu-calc($value, $query-width, $query-height);
#{$property}: vu-calc($value, $query-width, $query-height);

**Comment by seavor** [comment]
Yeah, it appears that Sass is typing the query width and height as strings instead of numbers like it is with the value, which throws off the strip-units function.
